<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1103</width>
    <height>528</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QScrollArea" name="scrollArea">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>271</width>
      <height>201</height>
     </rect>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="imgArea">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>269</width>
       <height>199</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="btLoad">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>240</y>
      <width>110</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Carregar</string>
    </property>
   </widget>
   <widget class="QSlider" name="slider">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>440</y>
      <width>1080</width>
      <height>30</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-200</number>
    </property>
    <property name="maximum">
     <number>200</number>
    </property>
    <property name="singleStep">
     <number>10</number>
    </property>
    <property name="pageStep">
     <number>30</number>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="sliderPosition">
     <number>0</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="lbLuz">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>410</y>
      <width>67</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Brilho</string>
    </property>
   </widget>
   <widget class="QCustomPlot" name="customPlot" native="true">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>0</y>
      <width>781</width>
      <height>391</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btValidacao">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>310</y>
      <width>110</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Validação</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btMakeHist">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>240</y>
      <width>110</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Histograma</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btEqualizar">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>310</y>
      <width>110</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Equalizar</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1103</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
